lib/Pinto/Action/Verify.pm

```diff
@@ -14,25 +16,110 @@ use Pinto::Util qw(throw);
 
 #------------------------------------------------------------------------------
 
-extends qw( Pinto::Action );
+extends qw( Pinto::Action::List );
```

Comment:
I think this would be the first Action that subclasses another one. That might be an indicator that some code needs to be refactored out into a role.